.ci/cico_build_and_deploy_custom_rh-che.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+set -x
+set -e
+set +o nounset
+
+/usr/sbin/setenforce 0
+
+export ORIGIN_CLIENTS_URL=http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/origin-clients-3.7.1-2.el7.x86_64.rpm
+export OPENSHIFT_RDU2C_URL=https://dev.rdu2c.fabric8.io:8443/
+export OC_VERSION=3.9.19
+
+function tagAndPushDocker() {
+  export RH_CHE_AUTOMATION_BUILD_TAG=rh-che-automated-build-$(git rev-parse --short HEAD)
+  export CHE_DOCKER_BASE_IMAGE=eclipse/che-server:nightly-centos
+  echo "che-docker-base-image:${CHE_DOCKER_BASE_IMAGE}"
+  docker pull ${CHE_DOCKER_BASE_IMAGE}
+  docker tag ${CHE_DOCKER_BASE_IMAGE} eclipse/che-server:local
+  rm -rf ./dockerfiles/che-fabric8/eclipse-che
+  mkdir ./dockerfiles/che-fabric8/eclipse-che
+  cp -r ./assembly/assembly-main/target/eclipse-che-fabric8-1.0.0-SNAPSHOT/eclipse-che-fabric8-1.0.0-SNAPSHOT/* ./dockerfiles/che-fabric8/eclipse-che/
+  bash ./dockerfiles/che-fabric8/build.sh || exit 1
+  docker rmi eclipse/che-server:local
```

Comment:
Removing the images free some space but may make your next build slower (because you need to download the layers once again). Do we really need to aggressively free space?

Reply:
It's not shared anyways. the duffy nodes get wiped at the end of each jenkins job, and as far as I know there is no central docker for `ci.centos.org`. We have just been discussing this issue.